### PR TITLE
CE-924

### PR DESCRIPTION
**Pull request by timmyquivy**
Increase height of InterwikiEdit data field

### CODE REVIEW

**Review by macbre — APPROVED** [review]

File: extensions/wikia/SpecialInterwikiEdit/SpecialInterwikiEdit.php [hunk 1]
```diff
@@ -370,7 +370,7 @@ function updateAction(action){
 	if ($from) $from = "WHERE iw_prefix like '". $db->escapeLike($from) . "%'";
```
Comment (WikiaTech) [inline extensions/wikia/SpecialInterwikiEdit/SpecialInterwikiEdit.php:370]: MAJOR Add curly braces around the nested statement(s). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
MINOR 2 statements were found on this line. Reformat the code to have only one statement per line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS122)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 58 issues
* MAJOR 31 major
* MINOR 23 minor
* INFO 4 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [SpecialInterwikiEdit.php#L19](https://github.com/Wikia/app/blob/39d97c28a572f1a3bbed793b5c0adc0b9fe55e6c/extensions/wikia/SpecialInterwikiEdit/SpecialInterwikiEdit.php#L19): Remove the parentheses from this "echo" call. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS2041)
1. MAJOR [SpecialInterwikiEdit.php#L52](https://github.com/Wikia/app/blob/39d97c28a572f1a3bbed793b5c0adc0b9fe55e6c/extensions/wikia/SpecialInterwikiEdit/SpecialInterwikiEdit.php#L52): Remove this commented out code. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)
1. MAJOR [SpecialInterwikiEdit.php#L54](https://github.com/Wikia/app/blob/39d97c28a572f1a3bbed793b5c0adc0b9fe55e6c/extensions/wikia/SpecialInterwikiEdit/SpecialInterwikiEdit.php#L54): Add curly braces around the nested statement(s). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
1. MAJOR [SpecialInterwikiEdit.php#L55](https://github.com/Wikia/app/blob/39d97c28a572f1a3bbed793b5c0adc0b9fe55e6c/extensions/wikia/SpecialInterwikiEdit/SpecialInterwikiEdit.php#L55): Add curly braces around the nested statement(s). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
1. MAJOR [SpecialInterwikiEdit.php#L85](https://github.com/Wikia/app/blob/39d97c28a572f1a3bbed793b5c0adc0b9fe55e6c/extensions/wikia/SpecialInterwikiEdit/SpecialInterwikiEdit.php#L85): Remove this unused "$wikiaDB" local variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)
1. MAJOR [SpecialInterwikiEdit.php#L159](https://github.com/Wikia/app/blob/39d97c28a572f1a3bbed793b5c0adc0b9fe55e6c/extensions/wikia/SpecialInterwikiEdit/SpecialInterwikiEdit.php#L159): Add curly braces around the nested statement(s). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
1. MAJOR [SpecialInterwikiEdit.php#L164](https://github.com/Wikia/app/blob/39d97c28a572f1a3bbed793b5c0adc0b9fe55e6c/extensions/wikia/SpecialInterwikiEdit/SpecialInterwikiEdit.php#L164): Remove this unused "$result" local variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)
1. MAJOR [SpecialInterwikiEdit.php#L183](https://github.com/Wikia/app/blob/39d97c28a572f1a3bbed793b5c0adc0b9fe55e6c/extensions/wikia/SpecialInterwikiEdit/SpecialInterwikiEdit.php#L183): The Cyclomatic Complexity of this function "wfSIWEEditInterwiki" is 23 which is greater than 20 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1541)
1. MAJOR [SpecialInterwikiEdit.php#L183](https://github.com/Wikia/app/blob/39d97c28a572f1a3bbed793b5c0adc0b9fe55e6c/extensions/wikia/SpecialInterwikiEdit/SpecialInterwikiEdit.php#L183): This function "wfSIWEEditInterwiki" has 243 lines, which is greater than the 150 lines authorized. Split it into smaller functions. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS138)
1. MAJOR [SpecialInterwikiEdit.php#L214](https://github.com/Wikia/app/blob/39d97c28a572f1a3bbed793b5c0adc0b9fe55e6c/extensions/wikia/SpecialInterwikiEdit/SpecialInterwikiEdit.php#L214): Remove this unused "$old_url" local variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 58 issues
* MAJOR 31 major
* MINOR 23 minor
* INFO 4 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [SpecialInterwikiEdit.php#L19](https://github.com/Wikia/app/blob/783cd430923cb3d210093b9cab6221d11d5fcaa1/extensions/wikia/SpecialInterwikiEdit/SpecialInterwikiEdit.php#L19): Remove the parentheses from this "echo" call. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS2041)
1. MAJOR [SpecialInterwikiEdit.php#L52](https://github.com/Wikia/app/blob/783cd430923cb3d210093b9cab6221d11d5fcaa1/extensions/wikia/SpecialInterwikiEdit/SpecialInterwikiEdit.php#L52): Remove this commented out code. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)
1. MAJOR [SpecialInterwikiEdit.php#L54](https://github.com/Wikia/app/blob/783cd430923cb3d210093b9cab6221d11d5fcaa1/extensions/wikia/SpecialInterwikiEdit/SpecialInterwikiEdit.php#L54): Add curly braces around the nested statement(s). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
1. MAJOR [SpecialInterwikiEdit.php#L55](https://github.com/Wikia/app/blob/783cd430923cb3d210093b9cab6221d11d5fcaa1/extensions/wikia/SpecialInterwikiEdit/SpecialInterwikiEdit.php#L55): Add curly braces around the nested statement(s). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
1. MAJOR [SpecialInterwikiEdit.php#L85](https://github.com/Wikia/app/blob/783cd430923cb3d210093b9cab6221d11d5fcaa1/extensions/wikia/SpecialInterwikiEdit/SpecialInterwikiEdit.php#L85): Remove this unused "$wikiaDB" local variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)
1. MAJOR [SpecialInterwikiEdit.php#L159](https://github.com/Wikia/app/blob/783cd430923cb3d210093b9cab6221d11d5fcaa1/extensions/wikia/SpecialInterwikiEdit/SpecialInterwikiEdit.php#L159): Add curly braces around the nested statement(s). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
1. MAJOR [SpecialInterwikiEdit.php#L164](https://github.com/Wikia/app/blob/783cd430923cb3d210093b9cab6221d11d5fcaa1/extensions/wikia/SpecialInterwikiEdit/SpecialInterwikiEdit.php#L164): Remove this unused "$result" local variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)
1. MAJOR [SpecialInterwikiEdit.php#L183](https://github.com/Wikia/app/blob/783cd430923cb3d210093b9cab6221d11d5fcaa1/extensions/wikia/SpecialInterwikiEdit/SpecialInterwikiEdit.php#L183): The Cyclomatic Complexity of this function "wfSIWEEditInterwiki" is 23 which is greater than 20 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1541)
1. MAJOR [SpecialInterwikiEdit.php#L183](https://github.com/Wikia/app/blob/783cd430923cb3d210093b9cab6221d11d5fcaa1/extensions/wikia/SpecialInterwikiEdit/SpecialInterwikiEdit.php#L183): This function "wfSIWEEditInterwiki" has 243 lines, which is greater than the 150 lines authorized. Split it into smaller functions. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS138)
1. MAJOR [SpecialInterwikiEdit.php#L214](https://github.com/Wikia/app/blob/783cd430923cb3d210093b9cab6221d11d5fcaa1/extensions/wikia/SpecialInterwikiEdit/SpecialInterwikiEdit.php#L214): Remove this unused "$old_url" local variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)

**Comment by ghost** [comment]
Tested and accepted. :medal_sports:

**Pull request merged**